tethys_apps/templates/tethys_apps/app_base.html

```diff
@@ -192,16 +211,54 @@
                     {%  endif %}
                   {% endblock %}
                   {% block settings_button_override %}
-                      {% if request.user.is_staff %}
-                      <div class="header-button settings-button">
-                        <a href="javascript:void(0);" onclick="TETHYS_APP_BASE.exit_app('{% url 'admin:index' %}tethys_apps/tethysapp/{{ tethys_app.id }}/change/');" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Settings"><i class="bi bi-gear"></i></a>
-                      </div>
-                      {%  endif %}
+                    {% if request.user.is_staff %}
+                    <div class="header-button settings-button">
+                      <a href="javascript:void(0);" onclick="TETHYS_APP_BASE.exit_app('{% url 'admin:index' %}tethys_apps/tethysapp/{{ tethys_app.id }}/change/');" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Settings"><i class="bi bi-gear"></i></a>
+                    </div>
+                    {%  endif %}
                   {% endblock %}
                   {% block exit_button_override %}
-                    <div class="header-button exit-button">
-                      <a href="javascript:void(0);" onclick="TETHYS_APP_BASE.exit_app('{% url 'app_library' %}');"data-bs-toggle="tooltip" data-bs-placement="bottom" title="Exit"><i class="bi bi-x"></i></a>
-                    </div>
+                    {% if not single_app_mode %}
```

Comment:
This looks like it's mostly duplicated from `tethys_portal/header.html`. Is there a way we could extract this into its own template (e.g. `user_menu.html`) and then just include it in both places?